### DISCUSSION TITLE
2Way Auth extra custom fields (TOTP)

### DISCUSSION THREAD

**Discussion by poudenes** [dani-garcia/vaultwarden#4028] [Ideas]
Hi ALl,

Would be great te have the option to add extra 2Way auth (TOTP). I have some services that have more then 1 TOTP to use. Now I create multiple items but that's not efficient.

Or if you want combine all TOTP into one item list.

**Comment by BlackDex** [comment]
Not possible from the Vaultwarden backend side. This needs to be done in all the clients, which this project does not maintain.

**Reply by poudenes** [reply]
So first have to ask Bitwarden to implement this and then it's possible?

**Reply by BlackDex** [reply]
That is the way to go.

Vaultwarden is only a backend implementation of Bitwarden which is compatible with the Bitwarden Clients.
